src/index/hgraph_index.cpp

```diff
@@ -662,27 +744,32 @@ HGraphIndex::calc_distance_by_id(const float* vector, int64_t id) const {
 }
 void
 HGraphIndex::add_one_point(const float* data, int level, InnerIdType inner_id) {
-    auto ep = this->entry_point_id_;
     MaxHeap result(allocator_);
+
+    InnerSearchParam param{
+        .ep_ = this->entry_point_id_,
```

Comment:
public fields do not need to end with `_`

Reply:
rename all public member soon